Source/Controllers/MainViewControllers/TableStructure/SPTableStructure.m

```diff
@@ -1495,6 +1495,15 @@ - (void)loadTable:(NSString *)aTable
 			[theField setObject:@"auto_increment" forKey:@"Extra"];
 		}
 
+        // Check for "generated always virtual | stored" and set Extra accordingly
+        if ([theField objectForKey:@"generatedalways"]) {
+            if ([[theField objectForKey:@"generatedalways"] isEqual:@"VIRTUAL"]) {
+                [theField setObject:@"VIRTUAL GENERATED" forKey:@"Extra"];
+            } else if ([[theField objectForKey:@"generatedalways"] isEqual:@"STORED"]) {
+                [theField setObject:@"STORED GENERATED" forKey:@"Extra"];
+            }
+        }
```

Comment:
I think we can use safeObjectForKey here and then remove the outer if because it's not really necessary.
safeObjectForKey is a helper I think @jamesstout made

Reply:
I'm agree with you.
But, in that method there are many blocks with the same kind of `if` using objectForKey. If I change the part of code I wrote, I think I should change all the others conditions too. As @Kaspik said above it would become bigger project. 
But I can do it, just tell me what, only the new code I wrote, all the method or nothing ?